Packages/com.unity.inputsystem/InputSystem/API/Input.cs

```diff
@@ -254,48 +258,224 @@ public static partial class Input
+                
+                case Inputs.Joystick_Trigger: return Joystick.current != null ? Joystick.current.trigger : null;
+
+                // TODO check do we need this for 2019.4?
+                // default: return null;
+            }
+
+            throw new ArgumentException($"Unexpected Inputs enum value '{input}'");
+        }
+        
+        // TODO consider overloads instead of one huge enum
+
         /// <summary>
-        /// Is the indicated control currently pressed.
+        ///     Is the indicated control currently pressed.
         /// </summary>
         /// <param name="input"></param>
         /// <returns></returns>
         /// <remarks>
-        /// This will look at all devices of the appropriate type (which will depend on the specified Inputs)
-        /// and return true if the control is currently pressed on any of them.
+        ///     This will look at all devices of the appropriate type (which will depend on the specified Inputs)
+        ///     and return true if the control is currently pressed on any of them.
         /// </remarks>
         public static bool IsControlPressed(Inputs input)
         {
-            throw new NotImplementedException();
+            return GetButtonControl(input)?.isPressed ?? false;
         }
 
         /// <summary>
-        /// True in the frame that the input was pressed.
+        ///     True in the frame that the input was pressed.
         /// </summary>
         /// <param name="input"></param>
         /// <returns></returns>
         /// <remarks>
-        /// This will look at all devices of the appropriate type (which will depend on the specified Inputs)
-        /// and return true if the control was actuated in the current frame on any of them.
+        ///     This will look at all devices of the appropriate type (which will depend on the specified Inputs)
+        ///     and return true if the control was actuated in the current frame on any of them.
         /// </remarks>
         public static bool IsControlDown(Inputs input)
         {
-            throw new NotImplementedException();
+            return GetButtonControl(input)?.wasPressedThisFrame ?? false;
         }
 
         /// <summary>
-        /// True in the frame that the input was released.
+        ///     True in the frame that the input was released.
         /// </summary>
         /// <param name="input"></param>
         /// <returns></returns>
         /// <remarks>
-        /// This will look at all devices of the appropriate type (which will depend on the specified Inputs)
-        /// and return true if the control was released in the current frame on any of them.
+        ///     This will look at all devices of the appropriate type (which will depend on the specified Inputs)
+        ///     and return true if the control was released in the current frame on any of them.
         /// </remarks>
         public static bool IsControlUp(Inputs input)
```

Comment:
Still got confused by the naming here (but maybe just me) as it doesn't imply it's temporal in nature.
I.e. this is going to return false the majority of the time, even when keys are not being pressed (and are 'up').